ui/client/components/graph/ProcessGraph.tsx

```diff
@@ -0,0 +1,54 @@
+import {DropTarget} from "react-dnd"
```

Comment:
Please add only one `/* eslint-disable i18next/no-literal-string */`

Reply:
it's not right way. ignores are temporary and should attract attention. every of 3 strings should be fixed (removed) in near future